docs/cluster-agent/GETTING_STARTED.md

```diff
@@ -0,0 +1,225 @@
+# Datadog Cluster Agent - DCA | Getting Started
+
+## Introduction
+
+This document aims to get you started upon the Datadog Cluster Agent. Refer to the [Datadog Cluster Agent - DCA | User Documentation](README.md) for more context about the Datadog Cluster agent. For more technical background, refer to the [Datadog Cluster Agent in Containerized environments](../../Dockerfiles/cluster-agent/README.md) documentation.
+
+## Step by step
+
+**Step 1** - The Datadog Cluster Agent (DCA) needs a proper RBAC to be up and running, first create its:
+
+* Service Account
+* Cluster Role
+* Cluster Role Binding
+
+Those can be found in the [Datadog Cluster Agent Rbac](../../Dockerfiles/manifests/cluster-agent/rbac/rbac-cluster-agent.yaml).
```

Comment:
Why the manifests are all stored inside `Dockerfiles` ? 😆

Reply:
/shrug

We should move that away I agree, but manifests are specific to Kubernetes so I guess it's not that bad.

Reply:
yeah it's fine for now I think. I'm creating a kubernetes folder for unrelated work, will move things there once the situation is clearer.